WinUIGallery/ControlPages/DesignGuidance/AccessibilityKeyboardPage.xaml

```diff
@@ -1,387 +1,673 @@
+                    <Button
+                        Grid.Row="1"
+                        Grid.Column="2"
+                        Click="MakeChartreuseButton_Click"
+                        Content="Chartreuse"
+                        ToolTipService.ToolTip="A greenish-yellow (Ctrl+G)">
+                        <Button.KeyboardAccelerators>
+                            <KeyboardAccelerator Key="G" Modifiers="Control" />
+                        </Button.KeyboardAccelerators>
+                    </Button>
+
+                    <TextBlock
+                        Grid.Row="2"
+                        Grid.ColumnSpan="4"
+                        Style="{StaticResource CaptionTextBlockStyle}"
+                        Text="Ctrl+R, Ctrl+B, and Ctrl+G trigger Red, Blue, and Chartreuse respectively" />
+                </Grid>
+            </core:ControlExample.Example>
+
+            <core:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;Grid ColumnSpacing="8" RowSpacing="8"&gt;
+&lt;Grid.RowDefinitions&gt;
+	&lt;RowDefinition Height="Auto" /&gt;
+	&lt;RowDefinition Height="Auto" /&gt;
+&lt;/Grid.RowDefinitions&gt;
+
+&lt;Grid.ColumnDefinitions&gt;
+	&lt;ColumnDefinition Width="Auto"/&gt;
+	&lt;ColumnDefinition Width="Auto"/&gt;
+	&lt;ColumnDefinition Width="Auto"/&gt;
+&lt;/Grid.ColumnDefinitions&gt;
+
+&lt;Rectangle x:Name="ColorRectangle" Fill="Red"
+	Height="30" RadiusX="4" RadiusY="4"
+	Grid.ColumnSpan="3"/&gt;
+
+&lt;Button Click="MakeRedButton_Click" Content="Red" Grid.Row="1"
+	ToolTipService.ToolTip="Shortcut: Ctrl+R"&gt;
+	&lt;Button.KeyboardAccelerators&gt;
+		&lt;KeyboardAccelerator Modifiers="Control" Key="R" /&gt;
+	&lt;/Button.KeyboardAccelerators&gt;
+&lt;/Button&gt;
+	
+&lt;Button Click="MakeBlueButton_Click" Content="Blue"
+	Grid.Row="1" Grid.Column="1"
+	ToolTipService.ToolTip="Shortcut: Ctrl+B"&gt;
+	&lt;Button.KeyboardAccelerators&gt;
+		&lt;KeyboardAccelerator Modifiers="Control" Key="B" /&gt;
+	&lt;/Button.KeyboardAccelerators&gt;
+&lt;/Button&gt;
+
+&lt;!-- Including the hotkey in a custom tooltip --&gt;
+&lt;Button Click="MakeChartreuseButton_Click" Content="Chartreuse"
+	Grid.Row="1" Grid.Column="1"
+	ToolTipService.ToolTip="A greenish yellow (Ctrl+G)"&gt;
+	&lt;Button.KeyboardAccelerators&gt;
+		&lt;KeyboardAccelerator Modifiers="Control" Key="G" /&gt;
+	&lt;/Button.KeyboardAccelerators&gt;
+&lt;/Button&gt;
+&lt;/Grid&gt;
+                </x:String>
+            </core:ControlExample.Xaml>
+            <core:ControlExample.CSharp>
+                <x:String xml:space="preserve">
+private void MakeRedButton_Click(object sender, RoutedEventArgs e)
+{
+    ColorRectangle.Fill = new SolidColorBrush(Colors.Red);
+}
+private void MakeBlueButton_Click(object sender, RoutedEventArgs e)
+{
+    ColorRectangle.Fill = new SolidColorBrush(Colors.Blue);
+}
+private void MakeChartreuseButton_Click(object sender, RoutedEventArgs e)
+{
+    ColorRectangle.Fill = new SolidColorBrush(Colors.Chartreuse);

```

Comment:
Interesting choice of color, why not green to use the 3 basic colors of computer graphics and also match the shortcut more closely?

Reply:
I chose chartreuse because it's a confusing color that warrants a custom tooltip (note that the XAML has a custom tooltip), which I use to demonstrate that you should include hotkey info in custom tooltips.

I'll leave a comment in the XAML, since that isn't obvious.

Reply:
That's a very great idea, thanks for clarifying and adding a comment for this!